bond/commands/upgrade.py

```diff
@@ -20,14 +20,15 @@ def get_branch_string(branch, target):
         return branch.replace("/", "-")
 
 
-def get_latest_version(target, branch):
+def get_s3_version(target, branch, depth=0):
     url = f"https://s3.amazonaws.com/bond-updates/v2/{target}/{branch}/versions_internal.json"
```

Comment:
```suggestion
    url = f"https://bond-updates.s3.amazonaws.com/v2/{target}/{branch}/versions_internal.json"
```
btw, we should change this as this old format is deprecated

Reply:
Indeed, feel free to make it in this PR or a separate one, either way sounds fine.